drools-wb-screens/drools-wb-drl-text-editor/drools-wb-drl-text-editor-backend/pom.xml

```diff
@@ -143,6 +143,17 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
@psiroky that is a great elaboration! It is perfectly clear, it's just difficult to keep it all in mind all the time. It would be best if you turn this into a wiki page in droolsjbpm-build-bootstrap, wdyt?

Just a single note: it is a good idea to have `src/test/resources/logback-test.xml` so that logging works during test execution.